osbuild/util/mnt.py

```diff
@@ -87,7 +98,9 @@ def umount(self):
             # cases `target is busy` error has been spotted.
             # Calling  `sync` does not hurt so we keep it for now.
             subprocess.run(["sync", "-f", target], check=True)
-            subprocess.run(["umount", target], check=True)
+            # NOTE: unmounting fails if the mountpoint has been remounted
+            # this is a workaround to ignore the message.
+            subprocess.run(["umount", target], check=not self.remount)
```

Comment:
Is there actually any value to umounting a remounted mount? Maybe we should run it only if `not self.remount`?